### ISSUE TITLE
Alternative Mailable Syntax not working with blade components

### ISSUE THREAD

**Issue by kskrlinnorth2** [laravel/framework#44569]
- Laravel Version: 9.35.1
- PHP Version: 8.1.0

### Description:
I try to use this alternative mailable syntax with blade components instead of blade directives in my emails, but it fails, outputting plain text:
![image](https://user-images.githubusercontent.com/61736280/195555040-5750c121-e649-49ca-a131-6db97bd6db5e.png)

Same issue with published laravel-mail and with removed vendor/mail

### Steps To Reproduce:

Mail class:
```
namespace App\Mail;

use Illuminate\Bus\Queueable;
use Illuminate\Mail\Mailable;
use Illuminate\Mail\Mailables\Content;
use Illuminate\Queue\SerializesModels;
use Illuminate\Mail\Mailables\Envelope;

class Test extends Mailable
{
    use Queueable, SerializesModels;

    public function envelope()
    {
        return new Envelope(
            subject: 'Test Email',
        );
    }

    public function content()
    {
        return new Content(
            markdown: 'emails.test',
            with: [
                'url' => '#',
            ],
        );
    }
}
```

Mail template:
```
<x-mail::message>
# This is test
 
Lorem ipsum dolor sit amet!
 
<x-mail::button :url="$url">
Click me
</x-mail::button>
 
Thanks,<br>
{{ config('app.name') }}
</x-mail::message>
```

**Comment by kskrlinnorth2** [comment]
My template file is missing .blade 🤦‍♂️

**Comment by kskrlinnorth2** [comment]
I answered on my own issue, my email template was missing .blade; so it was named test.php instead of teast.blade.php so it was not rendered properly. 

Now everything works fine with new mail syntax and blade components instead of blade directives (both works actually since we have old emails still using blade directives and new ones using blade components).

I did nothing extra in service provider or anywhere else, just updated framework to latest version which supports this new mail syntax.
